dbaas/logical/models.py

```diff
@@ -321,7 +323,9 @@ def delete(self, *args, **kwargs):
                     created_at=self.created_at
                 )
             except Exception, err:
-                LOG.error('Erro ao criar o database history para "o database {}: {}'.format(self.id, err))
+                LOG.error(
```

Comment:
Message in Portugues. I konw, it was this way, before you apply PEP8.

Reply:
Done :D